wagtail/admin/templates/wagtailadmin/notifications/updated_comments.txt

```diff
@@ -19,5 +19,13 @@
  - "{{ comment.text|safe }}"
 {% endfor %}{% endif %}
 
+{% if replied_comments %}
+{% trans "New replies" %}
+{% for thread in replied_comments %}
+{% trans 'New replies to:' %} - {{ thread.comment.text }}
```

Comment:
Could we remove the hyphen here? I think the colon is enough

Reply:
Yep, will do, thanks